.github/workflows/pre-submit.yaml

```diff
@@ -10,16 +11,21 @@ on:
       - master
       - main
       - release-*
-jobs:
-  test-and-build:
+jobs: # jobs to run
+  build:
     name: Test and build PRs
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-20.04 # # https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#choosing-github-hosted-runners
     steps:
     - name: Checkout code
-      uses: actions/checkout@v2
+      uses: actions/checkout@v3
       with:
         fetch-depth: 0
 
+    - name: Set up Go
```

Comment:
we don't need Go here 🤷🏼‍♂️

Reply:
Don't we need to `setup Go` to build our operators or it applies only to NHC and SNR?